ui/src/main/java/edu/wpi/grip/ui/MainWindowController.java

```diff
@@ -59,6 +66,7 @@
   private Pane aboutPane;
   @FXML
   private StatusBar statusBar;
+  private Label elapsedTimeLabel; // TODO improve the layout of this label
```

Comment:
This label isn't centered vertically in the status bar, and it separates the start/stop button from the "`Pipeline STATE`" text.